### PR TITLE
Unification the rendering of flyout columns

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description

The rendering values of the Mitre and Security operations flyout columns are changed to the same rendering of the events table, as it is the same rendering and can be handled all from the same place. 
And the rendering of the security operations requirements column is fixed.
 
### Issues Resolved

- #6844

### Evidence

Security operations

image
image
image
image
image

Mitre

image

### Test

1. Navigate to the different modules of Security operations > Control
2. Open flyouts 
3. Observe the following parameters:
- The agent.id and agent.name are links if they are not the agent 000 that redirects to the agent view
- The requirements if more than 1 must be separated by , 
- the rules must be a link that redirects to the rule.

1. Navigate to Mitre > Framework
2. Open flyout
3. Take into account the following parameters
- The agent.id and agent.name are links if they are not agent 000 that redirect to agent view
- The requirements if they are more than 1 must be separated by , 
- the rules must be a link that redirects to the rule.
- The technique must be a link that redirects to the technique.

1. Navigate to sameApplication > Events
2. Check that the links in the table work

1. Navigate to Threat Hunting > Dashboard
2. Check that the links in the table work: 
- Agent.id
- Agent.name
- rule.mitre.id
- rule.id

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [x] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by jbiset — APPROVED** [review]
## CR: :green_circle: 

## Test UI: :green_circle: 

### Security operations

image

image

image

image

image

**Redirect rule 503**

image


### Mitre

image

**Redirect technique**

image

**Comment by Desvelao** [comment]
# Tests

:green_circle: Security operations
image
image
image
image
image

:green_circle: MITRE ATTACK
image
image

**Review by Desvelao — DISMISSED** [review]
LGTM!

**Review by jbiset — APPROVED** [review]
LGTM

**Comment by Desvelao** [comment]
We found a problem with the redirection of the `Techniques` on the MITRE ATT&CK detail flyout. This will be solved on this pull request.

**Review by jbiset — APPROVED** [review]
Reviewed new changes

## CR: :green_circle: 

## Test UI: :green_circle: 

### Evidence

[changes_evidence.webm](https://github.com/user-attachments/assets/79bffa28-a622-42e3-bc5d-6ea5adeabbc1)

File: plugins/main/docs/wz-link.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+## Link compoenent
```
Comment (Desvelao) [inline plugins/main/docs/wz-link.md:1]: **nitpick**: typo

Comment (yenienserrano) [reply plugins/main/docs/wz-link.md]: Solved [75f18ba](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6848/commits/75f18ba13cb06226122981d2ecb0dfe421fcf6c0)

**Review by jbiset — APPROVED** [review]
LGTM

**Comment by Desvelao** [comment]
LGTM!

**Pull request merged**